backend/judging-algorithm/data/test/rotationTimes-controller.test.ts

```diff
@@ -0,0 +1,73 @@
+import db from './db.js'
+import controller from '../controllers/rotationTimes-controller.js'
+import { mockResponse, testCreateRotationTimeRequest } from './test-constants.js';
+
+beforeAll(async () => await db.connectDatabase())
+afterAll(async () => {
+    await db.clearDatabase();
+    await db.closeDatabase();
+})
+
+// create rotation time
```

Comment:
nit: all page comments can be deleted. and restructure test names to 
describe("Component") , it ("behavior") style here and in all test suites